src/hooks/useCart/useCart.tsx

```diff
@@ -288,18 +276,61 @@ export const useCart = (
         }
       }
     };
+    complete();
+  }, [cart, ids, selectedIdType, authKey, endpoint, features?.apiVersion]);
+
+  /**
+   * Handles the checking out of the cart.
```

Comment:
Will be good to add that if there's a need to confirm payment, then the other `completeCheckout` must be called